### PR TITLE
EOS-19375:CORTXCLI : Show and update of s3accesskey is not working as expected

### PR DESCRIPTION
**Pull request by rohitkolapkar**
# Backend

## Problem Statement
<pre>
Story Ref (if any):
</pre>
[EOS-19375](https://jts.seagate.com/browse/EOS-19375)
## Unit testing on RPM done
<pre>
No
</pre>
## Problem Description
<pre>
After login using s3 acoount on cortxcli, Show command of s3accesskey is returning user_name as "root" and 
expected username is logged in s3 account username.
</pre>
## Solution
<pre>
1. Help text changed for s3accesskeys command and its sub commands on cortxcli.
2. Changed the user_name from root to s3username in case of s3user request for accesskeys.
</pre>
## Unit Test Cases
<pre>
1. On cortxcli, login with s3 account
2. Fire the below command
s3accesskeys show -f json
3. Observe the user_name in the o/p it should be same as logged in s3 user_name
s3username 1

</pre>
Signed-off-by: rohitkolapkar <rohit.j.kolapkar@seagate.com>

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: csm/core/controllers/s3/access_keys.py [hunk 1]
```diff
@@ -69,6 +69,11 @@ async def get(self):
             resp = await self._service.list_access_keys(self._s3_session, **request_url_data)
             filtered_keys = [k for k in resp['access_keys'] if k['access_key_id'] not in tmp_keys]
             resp['access_keys'] = filtered_keys
+            # change the user_name, root to s3username in case of s3user request for accesskeys
+            if request_url_data.get('user_name') is None:
```
Comment (pawankumarsrivastava) [inline csm/core/controllers/s3/access_keys.py:73]: Move all the static strings to const.py

File: csm/cli/schema/s3accesskeys.json [hunk 2]
```diff
@@ -140,20 +140,20 @@
...
           "type": "str",
-          "help": "S3 IAM user name for which access key needs to be updated.",
+          "help": "S3 IAM user name for which status of access key needs to be updated.",
           "params": true
         },
         {
           "flag": "access_key_id",
           "type": "str",
-          "help": "Access key id to be updated."
+          "help": "Access key id for which status need to be updated"
```
Comment (pawankumarsrivastava) [inline csm/cli/schema/s3accesskeys.json:156]: need -> needs. Make sure that the line ending are consistent. Use full stop wherever required.

**Review by pawankumarsrivastava — CHANGES_REQUESTED** [review]

**Comment by pawankumarsrivastava** [comment]
@rohitkolapkar Please always write Problem Statement and description. `Refer to story` will not work. There is a reason for this template.

**Review by soniyamoholkar — APPROVED** [review]

File: csm/cli/schema/s3accesskeys.json (hunk 2 above)
Comment (rohitkolapkar) [reply csm/cli/schema/s3accesskeys.json:156]: sure @pawankumarsrivastava

**Review by avrnv — APPROVED** [review]

File: csm/core/controllers/s3/access_keys.py (hunk 1 above)
Comment (rohitkolapkar) [reply csm/core/controllers/s3/access_keys.py]: I have made the suggested changes @pawankumarsrivastava

File: csm/cli/schema/s3accesskeys.json (hunk 2 above)
Comment (rohitkolapkar) [reply csm/cli/schema/s3accesskeys.json]: I have updated the changes @pawankumarsrivastava

**Review by kupranay — APPROVED** [review]

**Review by pawankumarsrivastava — APPROVED** [review]

**Comment by kupranay** [comment]
run mini-provisioning

**Comment by kupranay** [comment]
ok to test

**Comment by rohitkolapkar** [comment]
Mini Provisioning is failing due to decryption issue. Script is still trying to decrypt with older key.
image

**Comment by rohitkolapkar** [comment]
RPM build is successful. Log attached.
[console output.txt](https://github.com/Seagate/cortx-manager/files/6345001/console.output.txt)

**Comment by kupranay** [comment]
ok to test

**Pull request merged**